qiskit/circuit/quantumcircuit.py

```diff
@@ -887,25 +887,6 @@ class QuantumCircuit:
 
        assert qc.size() == 19
 
-    A particularly important circuit property is known as the circuit :meth:`depth`.  The depth
-    of a quantum circuit is a measure of how many "layers" of quantum gates, executed in
-    parallel, it takes to complete the computation defined by the circuit.  Because quantum
-    gates take time to implement, the depth of a circuit roughly corresponds to the amount of
-    time it takes the quantum computer to execute the circuit.  Thus, the depth of a circuit
-    is one important quantity used to measure if a quantum circuit can be run on a device.
```

Comment:
This paragraph might be worth preserving? It's helpful to mention we have the method `depth`.

Reply:
Ah, I thought we were removing entirely. I  can move this part of the explanatory info to the description in https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.QuantumCircuit#depth like Jake suggested

Reply:
Drafted [here](548fa8e4199bc7085633dbdbc88fe88f57b29ecb)